core/custom_field_api.php

```diff
@@ -1269,12 +1269,31 @@ function print_custom_field_input( array $p_field_def, $p_bug_id = null ) {
 	global $g_custom_field_type_definition;
 	if( isset( $g_custom_field_type_definition[$p_field_def['type']]['#function_print_input'] ) ) {
 		call_user_func( $g_custom_field_type_definition[$p_field_def['type']]['#function_print_input'], $p_field_def, $t_custom_field_value );
+		print_hidden_input( custom_field_presence_field_name( $p_field_def['id'] ), '1' );
 	} else {
 		trigger_error( ERROR_CUSTOM_FIELD_INVALID_DEFINITION, ERROR );
 	}
 }
 
 /**
+ * Constructs the name of the field that used as a flag that a custom field is present on the form.`
```

Comment:
Suggested rewording:
Constructs the name of a field used as a flag to indicate that a custom field is present on the form